### PR TITLE
Exercise1

### PR DESCRIPTION
**Pull request by SiarheiKhmalEY**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                stringList.add(bufferedReader.readLine());
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("The file was not read");
+        }
+        return stringList;
+    }
+
+    public static void writeToFile(List<String> wordList) {
+        try (BufferedWriter bufferedWriter = Files.newBufferedWriter(Path.of(OUTPUT))) {
```
Comment (jaroslawwoloch) [inline src/Main.java]: write to sysout

File: src/Main.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            AtomicInteger id = new AtomicInteger();
+            bufferedWriter.write("ID Word" + "\n");
+            wordList.forEach(word -> {
+                try {
+                    bufferedWriter.write(id.getAndIncrement() + "  " + word + "\n");
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
```
Comment (jaroslawwoloch) [inline src/Main.java]: in real application don't throw Exception or RuntimeException this is a bad practice

**Review by jaroslawwoloch — CHANGES_REQUESTED** [review]
remove idea project files and generated files

**Review by jaroslawwoloch — APPROVED** [review]

**Review by jaroslawwoloch — CHANGES_REQUESTED** [review]
remove idea project file from repository

**Comment by SiarheiKhmalEY** [comment]
> remove idea project file from repository

Already done. Thank You for the advice. I didn't check file .gitnore

**Review by jaroslawwoloch — APPROVED** [review]

**Pull request merged**